ztp/gitops-subscriptions/argocd/example/image-based-upgrades/README.md

```diff
@@ -0,0 +1,86 @@
+## Image-Based Upgrades (IBU)
+This directory contains examples of generating resources required for Image Based Upgrades (IBU) utilizing the [Life Cycle Agent operator](https://github.com/openshift-kni/lifecycle-agent). These examples define policies to automate image-based upgrades, ensuring seamless deployment across managed clusters through Gitops.
+
+
+### Prerequisites
+
+Before using the IBU examples, ensure that the following namespaces have been created:
+
+- `ztp-common`: The root policy will be created in this namespace. If you use another name for the `common` namespace, please remember to update the `example-oadp-policy.yaml` policy.
+- `openshift-adp`: The ConfigMap containing the related OpenShift API for Data Protection (OADP) Custom Resources (CRs) will be copied to this namespace on the applicable spoke cluster(s).
+
+### Setup ArgoCD Application
+
+To deploy the IBU examples, you can use the existing [policies-app](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/deployment/policies-app.yaml), which is also used for deploying DU profile policy examples. Refer to the [ReadMe](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md) section "Preparation of Hub cluster for ZTP" for detailed instructions on setting up the ArgoCD policies application.
+
+Ensure that your Git repository, which will be used with the ArgoCD policies application, contains a directory structured as follows:
+
+```plaintext
+├── ...
```

Comment:
I guess you missed the source-crs dir as the kustomization.yaml refer to in line 34 ?

Reply:
I initially had included it, however, since we are providing the `source-crs/ibu/OpenShift.yaml` as part of the ztp container, I thought it might be redundant and mislead the user into creating a `source-crs` directory and copying the `OpenShift.yaml` file there. I can add it back - wdyt?

Reply:
its okay, may be just mentioned that source-crs/ibu/OpenShift.yaml dir must be in the same dir where kutomization.yaml located. Otherwise it will not work

Reply:
I added it back with a statement clarifying that the `source-crs/ibu/OpenShift.yaml` is provided in the ztp image and that the user should ensure the appropriate directory structure pertaining to the kustomization.yaml file location.